elements/pfe-jump-links/demo/index.html

```diff
@@ -60,14 +60,21 @@ <h1><code>pfe-jump-links</code></h1>
       position: sticky;
       top: 0;
     }
-
-
+    .special {
+      --pfe-jump-links-nav--offset: 800;
```

Comment:
@Djfaucette - Can we add this variable to a section in the README and explain how to use it and how if both the offset attribute and CSS property are present, the offset attribute is used?

Reply:
Yes and yes.